### ISSUE TITLE
To translate into another language, what should I do?

### ISSUE THREAD

**Issue by PlumpMath** [EliasVincent/whisper-subtitles-webui#2]
Hello, Thank you for sharing.

If I want to translate into a language other than English, where should I insert the option code? 

Also, when setting options for saving files or adding subtitles to videos, I encounter an error at the final saving point, stating that the file does not exist or there is no access to the tmp directory. Even if I set the path to the current folder or the user folder, the error still occurs. However, if I change the path, the srt file is successfully generated. 

Your app is so neat and great. Thank you.

**Comment by EliasVincent** [comment]
Hello there!

> If I want to translate into a language other than English, where should I insert the option code?

That's sadly [a limitation by Whisper](https://platform.openai.com/docs/guides/speech-to-text/translations). For whatever reason, they only trained their thing to translate to English only..

> stating that the file does not exist or there is no access to the tmp directory.

Which OS are you using?

**Comment by PlumpMath** [comment]
I have tested it on RHEL Linux + Nix and macOS + Nix operating systems, excluding Windows (haha, I realized while running your app that my computer with RHEL installed supports CUDA, but it's not NixOS, and the NixGL-related configuration doesn't provide proper CUDA Torch performance. Thanks for that!). It could be a problem with Nix Python Gardio, but I'm not entirely sure. I'm a developer who has just started using AI modules, and I wanted to find a quick solution by asking this question. By the way, it's a bit disappointing that it's only translated in English. I saw on another OpenAI-Whisper web UI that translation is possible, and I started using Whisper because of your beautiful implementation. In the past, I was able to set up an automatic macro using packages like Emacs macros and Google Translate. However, I was wondering if there is a way to directly interpret SRT files with OpenAI? It's a bit off-topic from this issue, but I would like to try implementing it using Python and OpenAI for more accurate quality. Thank you.

**Comment by PlumpMath** [comment]
Hmm, I found one thing. `dl_translate` ? It seems to be in another web UI module. I did some research, and it came up right away. I guess I'll just stick to using Emacs as before. It's a hassle. Anyway, thank you. Also, I discovered that there is a Mac-optimized version called 'whisper-cpp' which is widely used in the Korean internet community. It's a web UI that requires an Nvidia card to build. Anyway, I hope that within this year, the problem of only getting results in English will be solved, and we can quickly have a world where everyone can converse in their own language.
